en/docs/tutorials/create-rest-api-component.md

```diff
@@ -0,0 +1,299 @@
+5. Click **Create**.
+
+The `Statistics` REST API opens on a separate page.
+
+### Design the REST API
+
+Let's design the REST API as follows:
+
+1. To open the REST API component in the Web Editor, click **Edit with VSCode Online**.
+
+    Then click **Open Link in New Tab** in the message that appears to indicate that the code server is ready.
+
+2. On the left navigator, click **service.bal**  to open the low-code diagram of your REST API component.
+
+3. Add a new construct and an API resource as follows:
+
+    1. Click on the first **+** icon to add an HTTP construct. Select the **Service** construct type,
+
+        ![Add service construct](../assets/img/tutorials/rest-api/add-service-construct.png){.cInlineImage-half}
+
+        Select HTTP as the service type, and enter the following information.
+
+        | **Field**             | **Value**             |
+        |-----------------------|-----------------------|
+        | **Service Base Path** | `COVID19-Statistics`  |
+        | **Listener Port**     | `9090`                |
+
+        Select the **Define Inline** check box, and then click **Save**.
+
+    2. Expand the construct you created by clicking on it. Then click the **EDIT** icon of the default `GET` resource within the construct.
+
+    3. Enter `stats/[string country]` in the **Path** field.
+
+    4. Click **Advanced** and in the **Return Type** field, enter `json|error` to get the return in `json` or `error1` format.
+
+    5. Click **Save API**.
+
+4. To connect to the COVID-19 API and retrieve data, follow the sub-steps below.
+
+    1. Click on the API resource you edited, and then click on the first **+** icon after the **Start** statement.
+
+    2. Click **Connector** and then select **Covid19 API**.
+
+    3. In the **Covid19 API Connection** window, enter `covid19Client` as the **Connection Name** and click **Continue to Invoke API**.
+
+    4. In the **Operation** drop-down list, select **getStatusByCountry** and enter details as follows in the other fields:
+
+        | **Field**                  | **Value**         |
+        |----------------------------|-------------------|
+        | **Country**                | `country`         |
+        | **Response Variable Name** | `statusByCountry` |
+
+    5. Click **Save**.
+
+    6. Now let’s extract the total case count from the response and store it in a variable. Follow this procedure:
+
+        1. Click the last **+** icon in the low-code diagram.
+
+        2. Under **Statements**, select **Variable** and enter details as follows:
+
+            | **Field**      | **Value**                     |
+            |----------------|-------------------------------|
+            | **Type**       | `int`                         |
+            | **Name**       | `totalCases`                  |
+            | **Expression** | `<int>statusByCountry.cases`  |
+
+        3. Click **Save**.
+
+5. To retrieve the population data, connect to the World Bank API as follows:
```

Comment:
Why are we using two connectors? Shall we simplify this ti use one connector?

Reply:
We used two connectors for the Service quick start in V1 as well. Wouldn't the scenario be too basic with just one?